### DISCUSSION TITLE
Test de la sirène Immax Neo 07504L zigbee 3.0 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#7] [Comments]
# Test de la sirène Immax Neo 07504L zigbee 3.0 - Haade.fr

Test et déballage de la sirène Immax Neo zigbee 3.0 07504L quels sont ses atouts ?

https://haade.fr/fr/blog/test-sirene-immax-neo-07504L-zigbee3-par-domadoo

**Comment by roudou** [comment]
Bonjour, 
Merci pour l'article. 
Je l'ai recu j'attend une compatibilités avec z2mqtt si vous avez des informations ou les traces d'un snif cela m'intéresse

**Reply by haade-administrator** [reply]
j'ai commencé à adapter le code pour ce module je rencontre encore des soucis et fait une demande sur github à cette adresse https://github.com/Koenkk/zigbee-herdsman-converters/issues/5455

**Reply by roudou** [reply]
Bonjour , j'ai un peux modifié ton code sur gît .
Peux tu me dire s'il est possible de faire d'autres son ?

**Reply by haade-administrator** [reply]
J'ai pas eu le temps ces deux derniers jours d'avancer par contre j'ai pas l'impression qu'il prenne en compte d'autre son que celui par défaut de plus le tableau fourni sur le github n'indique pas de modèles de sons

**Comment by jrbienvenu** [comment]
Bonjour,
Avez vous du nouveau sur l'intégration de ce device sur zigbee2mqtt ?
Merci

**Comment by Rikko69** [comment]
Bonjour,
Un petit UP également pour savoir si vous avez pu avancer sur l'intégration.
J'ai basculé sur jeedom de Zigbee à JeeZigbee, j'avais l'alarme auparavant sur ZigBee mais mal reconnue sur JeeZigBee, en commande j'ai uniquement "Dernière communication" et l'équipement est juste reconnu comme un routeur.

Merci!

**Comment by haade-administrator** [comment]
non je n'ai pas pu avancer sur l'intégration car j'ai été confronté à pas mal de soucis et vraisemblablement je ne suis pas le seul
